CHANGELOG.md

```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
+  * Tune UI alignment for nav bar notification icon, avatar image, issue label (#17438)
+  * Add appearance section in settings (#17433)
+  * Move key forms before list and add cancel button (#17432)
+  * chmod executables when copying to the docker (#17423)
+  * fix: remove deprecated `extendDefaultPlugins` method of svgo (#17399)
+  * Fix the click behavior for <tr> and <td> with [data-href] (#17388)
+  * Refactor update checker to use AppState (#17387)
+  * Improve async/await usage, and sort init calls in `index.js` (#17386)
+  * Use a variable but a function for IsProd because of a slight performance increment (#17368)
+  * Frontend refactor, PascalCase to camelCase, remove unused code (#17365)
+  * Update JS dependencies (#17357)
+  * Hide command line merge instructions when user can't push (#17339)
+  * Move session to models/login (#17338)
+  * Sync gitea app path for git hooks and authorized keys when starting (#17335)
+  * Make the Mirror Queue a queue (#17326)
+  * Add "Copy branch name" button to pull request page (#17323)
+  * Fix repository summary on mobile (#17322)
+  * Split `index.js` to separate files (#17315)
+  * Show direct match on top for user search (#17303)
+  * Frontend refactor: move Vue related code from `index.js` to `components` dir, and remove unused codes. (#17301)
+  * Upgrade chi to v5 (#17298)
+  * Disable form autofill (#17291)
+  * Improve behavior of "Fork" button (#17288)
+  * Open markdown image links in new window (#17287)
+  * Add hints for special Wiki pages (#17283)
+  * Move add deploy key form before the list and add a cancel button (#17228)
+  * Allow adding multiple issues to a project  (#17226)
+  * Add metrics to get issues by repository (#17225)
+  * Add specific event type to header (#17222)
+  * Redirect on project after issue created (#17211)
+  * Reference in new issue modal: dont pre-populate issue title (#17208)
+  * Always set a unique Message-ID header (#17206)
+  * Add projects and project boards in exposed metrics (#17202)
+  * Add metrics to get issues by label (#17201)
+  * Add protection to disable Gitea when run as root (#17168)
+  * Don't return binary file changes in raw PR diffs by default (#17158)
+  * Support sorting for project board issuses (#17152)
+  * Force color-adjust for markdown checkboxes (#17146)
+  * Add option to copy line permalink (#17145)
+  * Move twofactor to models/login (#17143)
+  * Multiple tokens support for migrating from github (#17134)
+  * unify issue and pr subtitles (#17133)
+  * Make Requests Processes and create process hierarchy. Associate OpenRepository with context. (#17125)
+  * Fix problem when database id is not increment as expected (#17124)
+  * Avatar refactor, move avatar code from `models` to `models.avatars`, remove duplicated code (#17123)
+  * Re-allow clipboard copy on non-https sites (#17118)
+  * refactor: move from io/ioutil to io and os package (#17109)
```

Comment:
Rm refactor:

Reply:
I vote just remove this from the changelog.